python/ql/lib/semmle/python/dataflow/new/internal/SummaryTypeTracker.qll

```diff
@@ -312,8 +322,8 @@ module SummaryFlow<Input I> implements Output<I> {
     |
       callable.propagatesFlow(input, output, true) and
       call = I::callTo(callable) and
-      nodeFrom = evaluateSummaryComponentStackLocal(callable, call, input) and
-      nodeTo = evaluateSummaryComponentStackLocal(callable, call, output)
+      nodeFrom = evaluateSummaryComponentStackLocal(callable, call, input, false) and
+      nodeTo = evaluateSummaryComponentStackLocal(callable, call, output, true)
     )
   }
```

Comment:
This predicate is not a store step, should it have the same treatment? (The commit title indicates maybe not.)

Reply:
Yes, it should have the same treatment, good question though.